package.json

```diff
@@ -9,6 +9,7 @@
     "lint:ts": "eslint --ext .ts,.js .",
     "lint:yaml": "yamllint --ignore=node_modules/**/*.yaml **/*.yaml",
     "test:e2e": "mocha \"tests/e2e/**/*.test.ts\"",
+    "test:semantic-mocha": "nyc --extension .ts mocha --config semantic-mocha/tests/.mocharc.js \"semantic-mocha/tests/**/*.test.ts\"",
```

Comment:
Should `semantic-mocha` have it's own `package.json` with this command?

Reply:
Nah